addons/medical_damage/functions/fnc_woundsHandlerVehiclehit.sqf

```diff
@@ -0,0 +1,37 @@
+#include "script_component.hpp"
+/*
+ * Author: Pterolatypus, LinkIsGrim
+ * Custom wound handler for vehicle hits and explosions, sends damage to a random hitpoint
+ *
+ * Arguments:
+ * 0: Unit That Was Hit <OBJECT>
+ * 1: Damage done to each body part <ARRAY>
+ * 2: Type of the damage done <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, [[0.5, "#structural", 1.5]], "vehiclehit"] call ace_medical_damage_fnc_woundsHandlerVehiclehit
+ *
+ * Public: No
+ */
+params ["_unit", "_allDamages", "_typeOfDamage"];
+TRACE_3("woundsHandlerVehiclehit",_unit,_allDamages,_typeOfDamage);
+
+// this should only trigger for hits to just structural
+if (count _allDamages > 1) exitWith {_this};
+
+// damage can sometimes be negative (why?)
+// damage to structural is low unless it's a very large explosion, in which case it is typically >= 1
+private _damageToApply = (abs (_allDamages select 0 select 0));
+
+private _newDamages = [];
+
+// hitpoints are randomized, more damage means more wounds in different body parts
+for "_i" from 1 to (_damageToApply * 6) do {
+    _newDamages pushBack [_damageToApply * 6, selectRandom ALL_BODY_PARTS, _damageToApply * 6]
```

Comment:
I understand the 6x on line 32, but not sure I get where the 6x on 33 is coming from?

Reply:
IIRC, scaling. Though looking at this again now I'd rather rewrite this loop.

Reply:
Remembered: damage here is only gotten from `#structural` hitpoint and only available in `0-1` range, typically very low. So it's multiplied by 6 to account for it representing damage to each limb, sorta. It feels right in game IMO.